src/components/views/post-list.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import './post-list.scss';
+
+class PostList extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const { posts } = this.props;
+        return (
+            <div>{posts.map(post => {
+                let date = new Date(post.date * 1000);
+
+                return (
+                    <article key={post.id} className="article-body">
+                        <header>
+                            <div className="article-date">{date.toDateString()}</div>
+                            <h1 className="article-title">{post.title}</h1>
+                            <div className="article-info">
+                                <a href="#">{post.author}</a> ● {post.tags.map(tag => {return <a href="#" key={tag.toString()}><span>{tag}</span></a>})}
```

Comment:
Sometime, don't be afraid to extract even such small things as Tag into component, because later more features can appear for them. E.g. we can have static tag (just link) and editable tag, with 'x' icon on it.